components/ai_chat/resources/page/components/delete_conversation_modal/index.tsx

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import Button from '@brave/leo/react/button'
+import Dialog from '@brave/leo/react/dialog'
+
+// Utils
+import { getLocale } from '$web-common/locale'
+
+// Hooks
+import { useAIChat } from '../../state/ai_chat_context'
+
+// Styles
+import styles from './style.module.scss'
+
+export default function DeleteConversationModal() {
+  // Context
+  const aiChatContext = useAIChat()
+
+  // Computed
+  const title = !aiChatContext.deletingConversationId
+    ? getLocale('conversationListUntitled')
```

Comment:
Couldn't this just be the empty string? We won't be showing the dialog, so it should be safe, right? If not, this should be covered by the fallback to `getLocale('conversationListUntitled')`

Reply:
Was using as a `Type` safety check here, but you are right it will fallback to `getLocale('conversationListUntitled')`. Updated this!